files/default/sql/ddl/updates/1.3.0.sql

```diff
@@ -40,3 +40,10 @@ CREATE TABLE IF NOT EXISTS `training_dataset_split` (
   KEY `training_dataset_id` (`training_dataset_id`),
   CONSTRAINT `training_dataset_fk` FOREIGN KEY (`training_dataset_id`) REFERENCES `training_dataset` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+ALTER TABLE `hopsworks`.`python_dep` ADD COLUMN `base_env` VARCHAR(45) COLLATE latin1_general_cs;
+ALTER TABLE `hopsworks`.`conda_commands` DROP FOREIGN KEY `FK_481_519`;
```

Comment:
`FK_481_519` is a random id generated by MySQL if you create a constraint without name. So there might be deployments out there that have a different name for the same fk.

The correct way of dropping this is to look up the name in the `INFORMATION_SCHEMA` - here is an example: https://github.com/logicalclocks/hopsworks-chef/blob/d70685ba5aea4f77689e42b65ffad3650d27d92e/files/default/sql/ddl/updates/1.0.0.sql#L221-L226

Reply:
I took the FK_481_519 from here: https://github.com/logicalclocks/hopsworks-chef/blob/1c401873137f1cb8d05e29a86f7c988ba6466abc/files/default/sql/ddl/1.3.0__initial_tables.sql#L180
So it seems to come from us. But I'll do what you say, to be on the safe side.